factory/alchemy.py

```diff
@@ -99,13 +99,13 @@ def _create(cls, model_class, *args, **kwargs):
         return cls._save(model_class, session, *args, **kwargs)
 
     @classmethod
-    def _save(cls, model_class, session, *args, **kwargs):
+    def _save(cls, model_class, sqlalchemy_session, *args, **kwargs):
```

Comment:
Why not pass the kwargs dict to `_save` instead of using variadic keyword arguments? That would entirely prevent name conflicts.

Reply:
Good shout. I've made the change — let me know if it matches your expectations.